internal/helm/plan.go

```diff
@@ -93,6 +93,12 @@ func (p *Plan) Execute() error {
 var upgrade = func(cfg env.Config) []Step {
 	var steps []Step
 	steps = append(steps, run.NewInitKube(cfg, kubeConfigTemplate, kubeConfigFile))
+
+	if cfg.ConvertV2Releases {
+		// The "helm" context is coming from the template
+		steps = append(steps, run.NewConvert(cfg, kubeConfigFile, "helm"))
```

Comment:
**question:** What is the `helm` context? Is there any documentation you can point us to?

Reply:
The helm context is just a generic name that is used in the kubeconfig template: https://github.com/mongodb-forks/drone-helm3/blob/master/assets/kubeconfig.tpl#L18

Basically there's only one context that is configured to point to whatever namespace is defined in the plugin config